sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -495,6 +646,22 @@ export class ServiceBusManagementClient extends ServiceClient {
     return { _response: response };
   }
 
+  /**
+   * Checks whether a given queue exists or not.
+   * @param queueName
+   */
+  async queueExists(queueName: string): Promise<boolean> {
+    log.httpAtomXml(`Performing management operation - queueExists() for "${queueName}"`);
+    try {
+      await this.getQueue(queueName);
+    } catch (error) {
+      if (error.code == "MessageEntityNotFoundError") {
+        return false;
+      }
```

Comment:
Shouldnt this throw the error if the error was not `MessageEntityNotFoundError`?
cc @ShivangiReja

Reply:
That's probably better.

I think .NET SDK should also update accordingly.

Reply:
Updated in this PR.